Lib/unittest/case.py

```diff
@@ -852,7 +852,8 @@ def assertAlmostEqual(self, first, second, places=None, msg=None,
         """Fail if the two objects are unequal as determined by their
            difference rounded to the given number of decimal places
            (default 7) and comparing to zero, or by comparing that the
-           between the two objects is more than the given delta.
+           difference between the two objects is more than the given 
```

Comment:
Thanks for the pull request @selotape!

The tests are failing because you have an extra space after the word "given".

![screen shot 2017-10-17 at 10 28 37 am](https://user-images.githubusercontent.com/878798/31676870-337400f4-b326-11e7-9d25-72aae7f66d8e.png)

Reply:
Thanks even more for the review @dianaclarke !

I fixed the extra space, as well as the same phrasing issue in `assertNotAlmostEqual`